auto-value-gson-extension/src/main/java/com/ryanharter/auto/value/gson/AutoValueGsonExtension.java

```diff
@@ -1098,7 +1146,7 @@ private static String proguardNameOf(TypeName typeName) {
     } else if (typeName instanceof TypeVariableName) {
       return "java.lang.Object";
     } else {
-      throw new UnsupportedOperationException("Unrecognized TypeName type: " + typeName);
+      throw new UnsupportedOperationException("unrecognised TypeName type: " + typeName);
```

Comment:
```suggestion
      throw new UnsupportedOperationException("Unrecognized TypeName type: " + typeName);
```
let's stick to American English as the rest of the codebase does, same everywhere else in the PR

Reply:
renamed